internal/controller/pkg/controller/options.go

```diff
@@ -49,6 +49,14 @@ type Options struct {
 	// injected to CRDs so that API server can make calls to the providers.
 	WebhookTLSSecretName string
 
+	// TLSServerSecretName is the Secret that will be mounted to provider Pods.
```

Comment:
Is it worth mentioning what this is for? (Webhooks, right?)

Reply:
correct, updated the comment